bears/haskell/HaskellLintBear.py

```diff
@@ -19,7 +21,9 @@ class HaskellLintBear:
     """
 
     LANGUAGES = {'Haskell'}
-    REQUIREMENTS = {DistributionRequirement(apt_get='hlint')}
+    REQUIREMENTS = {CabalRequirement(
+        package='ghc-mod', version='5.6.0'),
```

Comment:
For format of using CabalRequirement refer doctests of https://gitlab.com/coala/package_manager/blob/master/dependency_management/requirements/CabalRequirement.py